docs/web-apps/automated-testing/cypress.md

```diff
@@ -65,7 +65,7 @@ You can run `saucectl` locally via Docker ([Installation Requirements](https://d
     <tbody>
     <tr>
       <td rowspan='2'>9.5.3</td>
-      <td><b>macOS:</b> 11.0</td>
+      <td><b>macOS:</b> 11.00, 12</td>
```

Comment:
I believe macOS minor versions use single-digit zeros (i.e., 11.0). See https://en.wikipedia.org/wiki/MacOS_Big_Sur#Release_history.

Reply:
You are correct! However, Sauce does not 😓  The reason why I changed it is a bit more complicated. Sauce expects `mac(OS) 11.00` for mac 11, and for 12, it's `mac(OS) 12`. Semantically, neither `11.0` nor `11.00` are correct, because Sauce serves you a VM with the highest minor patched version for `11` or `12` (e.g. `12.3.1`). Going forward, we'll only let the user specify major versions.

The change in the doc now reflects reality more closely, with values in the config that the user specifies.